src/app/feature/stack/components/stack/stack.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { ModalController } from "@ionic/angular";
+
+export interface IStackConfig {
+  templateName: string;
+  title?: string;
+  showCloseButton?: boolean;
+}
+
+/** The logic for stack open/dismiss exists in the stack-navigation service */
+@Component({
+  selector: "plh-stack",
+  templateUrl: "./stack.component.html",
+  styleUrls: ["./stack.component.scss"],
+})
+export class StackComponent {
+  private _config: IStackConfig = {
+    templateName: "",
+    showCloseButton: true,
+  };
+  // Merge input values with defaults, ignoring any `undefined` values
+  @Input()
+  set config(config: IStackConfig) {
+    this._config = {
+      ...this._config,
+      ...Object.fromEntries(Object.entries(config).filter(([_, v]) => v !== undefined)),
```

Comment:
nit(non-blocking): angular input signals offer a nicer pattern than defining separate getter/setter input handlers (and intermediate variables), as they expose a `transform` operation that can be used to merge defaults with passed values:
https://angular.dev/guide/components/inputs#input-transforms